.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: go
```

Comment:
I know this is not mandatory but could we have the `---` that was there before? It's common/standard in all of our YAML files. For [an explanation why, see here](https://stackoverflow.com/a/50788318). If you feel strongly not to add it, then it's cool, you can leave it out.

Reply:
Done https://github.com/Intellection/vault-init/pull/1/commits/4d73f0b85c72f490f67ef5689df76b0313f59c99